qualtran/bloqs/arithmetic.py

```diff
@@ -208,7 +215,121 @@ def pretty_name(self) -> str:
         return "a gt b"
 
     def t_complexity(self):
-        # TODO Determine precise clifford count and/or ignore.
-        # See: https://github.com/quantumlib/cirq-qubitization/issues/219
-        # See: https://github.com/quantumlib/cirq-qubitization/issues/217
-        return TComplexity(t=8 * self.bitsize)
+        return t_complexity(LessThanEqualGate(self.bitsize, self.bitsize))
```

Comment:
is this true? please document how/why the counts of greaterthan is that of lessthanequal

In general, it's good to at least note these things so readers don't suspect it was a copy-paste error or something.

Reply:
I'm going to split this PR up, and get some input on the correct costs there.

Reply:
This was a bit loosy goosey > ~ <= and > constant ~ < constant was my assumption.

Reply:
Was this split into another PR? Do you need to merge it in? 

If no: please add a note to the costs

Reply:
Yeah sorry. I separated out into #381 and was going to get some feedback about the appropriate. Costs. I'll merge that  PR before this one.